rosette-scala/src/main/scala/coop/rchain/rosette/Location.scala

```diff
@@ -136,4 +136,103 @@ object Location {
 
       case LTLimbo => Ob.INVALID
     }
+
+  def printRep(loc: Location): String = {
+    val names = List("rslt",
+                     "trgt",
+                     "argvec",
+                     "env",
+                     "code",
+                     "ctxt",
+                     "self",
+                     "self-env",
+                     "rcvr",
+                     "monitor")
+
+    loc.genericType match {
+      case LTCtxtRegister(reg) =>
+        if (0 < reg && reg < Location.NumberOfCtxtRegs) {
```

Comment:
In the C++ implementation it is `0 <= reg`.

Reply:
Good catch, thanks!